pkgs/development/python-modules/pytest-rerunfailures/default.nix

```
@@ -1,23 +1,23 @@
-{ lib, buildPythonPackage, pythonOlder, fetchPypi, pytest, mock }:
+{ lib, pythonPackages, pythonOlder, fetchPypi, setuptools, pytest, pytestCheckHook, mock }:
 
-buildPythonPackage rec {
+pythonPackages.buildPythonPackage rec {
   pname = "pytest-rerunfailures";
-  version = "10.3";
+  version = "11.0";
+  format = "pyproject";
 
-  disabled = pythonOlder "3.5";
+  disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-2CRNeZ+Jpu215XMB3a6ztvENZpFjjVHoCzcTEVkuKMY=";
+    hash = "sha256-zuWeTm8Nc65j+6CZhlMUupuRW3GTSVQREPoBL/tu+xM=";
   };
 
-  buildInputs = [ pytest ];
+  nativeBuildInputs = [ setuptools ];
 
-  nativeCheckInputs = [ mock pytest ];
+  buildInputs = [ pytest ];
+  propagatedBuildInputs = with pythonPackages; [ packaging ];
```

```suggestion
  propagatedBuildInputs = [ packaging ];
```

Reference it without pythonPackages.